draft-ietf-core-yang-cbor-latest.md

```diff
@@ -1620,7 +1632,30 @@ application/yang-data+cbor; id=name:
 : The message payload of Content-Type 'application/yang-data+cbor' is encoded using a CBOR map.
   Each entry within the CBOR map contains the data node identifier (i.e., its
   namespace qualified name) and the associated instance-value.  Instance-values
-  are encoded using the rules defined in {{instance-encoding}}
+  are encoded using the rules defined in {{instance-encoding}}.
+
+
+For CBOR YANG documents making use of YANG SID identifiers only:
+
+application/yang-data+cbor; id=sid:
+: This Content-Type represents a CBOR YANG document containing one or multiple data node values.
+  Each data node is identified by its associated SID as defined in {{sid}}.
+
+: FORMAT: CBOR map of SID, instance-value
+
+: The message payload of Content-Type 'application/yang-data+cbor' is encoded using a CBOR map.
```

Comment:
It feels a bit odd that a parameter value such as `sid` defines the entire message format from scratch. Wouldn't it be better to specify the CBOR structure once and then describe how the `id` parameter further constrains the CBOR structure in a certain way?

Reply:
Yes, I'm a bit uncomfortable with this section now, too.
So after pushing the above fixes I'll have a go at refactoring it.